internal/server/ranking/ranking_test.go

```diff
@@ -111,6 +111,16 @@ func TestSeriesByRank(t *testing.T) {
 				{ImportName: "USFEMA_NationalRiskIndex", Unit: "FemaNationalRiskScore"},
 			},
 		},
+		{
+			[]*pb.SourceSeries{
+				{ImportName: "EarthquakeUSGS_Agg", MeasurementMethod: "GridWeightedPearson", ObservationPeriod: "P1M"},
+				{ImportName: "EarthquakeUSGS_Agg", MeasurementMethod: "GridWeightedPearson", ObservationPeriod: "P1Y"},
```

Comment:
nit: maybe throw in some non-EarthquakeUSGS_Agg and P1D too.

Reply:
Done.